### PR TITLE
DGC-30: adding user registration

### PR DESCRIPTION
**Pull request by mikemadison13**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -114,6 +114,3 @@ nbproject/*
 *.gz
 *.sql
 *.zip
-
```
Comment (aweingarten) [inline .gitignore]: This should be restored.

File: hooks/common/post-code-deploy/post-code-deploy.sh [hunk 2]
```diff
@@ -16,10 +16,9 @@ source_branch="$3"
 deployed_tag="$4"
 repo_url="$5"
 repo_type="$6"
-drush_alias=${site}'.'${target_env}
 
-# Load utility functions.
-. `dirname $0`/../capitalcamp-functions.sh
```
Comment (aweingarten) [inline hooks/common/post-code-deploy/post-code-deploy.sh]: Remove capitalcamp-functions.sh from codebase.  They have been replaced by built-in cloudhooks.

File: project.yml [hunk 3]
```diff
@@ -31,4 +31,14 @@ target-hooks:
         dir: '${docroot}/profiles/contrib/lightning'
         command: 'npm run install-libraries'
 import: null
+modules:
+    ci:
+        enable: {  }
+        uninstall: [acquia_connector, shield]
+    deploy:
+        enable: [acquia_connector, shield]
```
Comment (aweingarten) [inline project.yml]: Need to remove shield.

**Comment by mikemadison13** [comment]
I'll clean up what @aweingarten  noted. The other big issue that keeps causing the build to fail now is that the CMI is causing fatal errors because of missing database tables. So I also need to dig into that.

**Pull request closed without merging**